strategic-entrepreneurship-journal.csl

```diff
@@ -0,0 +1,237 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-GB">
+  <info>
+    <title>Strategic Entrepreneurship Journal</title>
+    <id>http://www.zotero.org/styles/strategic-entrepreneurship-journal</id>
+    <link href="http://www.zotero.org/styles/strategic-entrepreneurship-journal" rel="self"/>
+    <link href="https://onlinelibrary.wiley.com/page/journal/1932443x/homepage/forauthors.html" rel="documentation"/>
+    <author>
+      <name>Aku Valtakoski</name>
+      <email>aku.valtakoski@tkk.fi</email>
+    </author>
+    <contributor>
+      <name>Fabio Motoki</name>
+    </contributor>
+    <category citation-format="author-date"/>
+    <category field="social_science"/>
+    <issn>1932-443X</issn>
```

Comment:
ISSN: 1932-4391
E-ISSN: 1932-443X
Please add the ISSN and relabel the one you have with eissn.